pkg/virt-launcher/virtwrap/network/common.go

```diff
@@ -142,47 +149,97 @@ func (h *NetworkUtilsHandler) UseIptables() bool {
 
 	return true
 }
-func (h *NetworkUtilsHandler) IptablesNewChain(table, chain string) error {
-	iptablesObject, err := iptables.New()
+
+func (h *NetworkUtilsHandler) ConfigureIpv6Forwarding() error {
+	_, err := exec.Command("sysctl", "net.ipv6.conf.all.forwarding=1").CombinedOutput()
+	return err
+}
+
+func (h *NetworkUtilsHandler) Ipv4NatEnabled() bool {
+	lsmod, _ := exec.Command("lsmod").CombinedOutput()
+	lsmodString := string(lsmod)
+	if !strings.Contains(lsmodString, "iptable_nat") && !strings.Contains(lsmodString, "nf_nat_ipv4") {
+		log.Log.Errorf("no ipv4 support")
+		return false
+	}
+	return true
+}
+
+func (h *NetworkUtilsHandler) Ipv6NatEnabled() bool {
+	lsmod, _ := exec.Command("lsmod").CombinedOutput()
```

Comment:
Done. And all `phase1` including the iptables/nftables configuration is done in the virt-handler anyway since [Configure pod network for launcher from virt-handler](https://github.com/kubevirt/kubevirt/pull/2837) was merged.